frontend/src/style/landing.scss

```diff
@@ -0,0 +1,29 @@
+$blue: #3248b4;
+$green: #397c29;
+
+.landing-page {
```

Comment:
This is exactly what I mean by wrapping. Use it for imageUploader! ⬆️

Reply:
Done!